### PR TITLE
adds Architectural Decision Records approach to GLAM's documentation

### PR DESCRIPTION
**Pull request by hamilton**
We're considering following the ADR methodology for noting big decisions. This entails:

- `/docs/adr/` – add an ADR here.
- `/scripts/update-adrs.js` – run this to re-compile the ADR index file.

### CODE REVIEW

File: scripts/update-adrs.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+To contribute a new ADR,
+
+1. copy \`template.md\` into a new file in this directory in the format
+\`nnnn-title-with-dashes.md\`. The \`nnnn\` should be the next number up from
+the highest number in the ADRs in this directory.
+2. fill out the template.
+3. once done, run node scripts/update-adrs to generate a new index.
+4. file a PR with these changes as a branch in the GLAM repository.
+
+## accepted ADRs
```
Comment (robhudson) [inline scripts/update-adrs.js]: Can we capitalize this?

**Comment by robhudson** [comment]
This is great!

**Review by robhudson — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
   [f11e979](https://github.com/mozilla/glam/commit/f11e9793b4725d2bfc6ebd222d481555918b1c24)
 - Add last updated model and API endpoint
   ([#743](https://github.com/mozilla/glam/pull/743))
+- Add `/docs/adr/` (a directory of Architectural Decision Records)
```
Comment (hamilton) [inline CHANGELOG.md:16]:
```suggestion
- Add `/docs/adr/` (a directory of Architectural Decision Records) ([#854](https://github.com/mozilla/glam/pull/854/))
```

File: scripts/update-adrs.js (hunk 1 above)
Comment (hamilton) [inline scripts/update-adrs.js]:
```suggestion
## Accepted ADRs
```

**Pull request merged**